### ISSUE TITLE
Where do I get libStreamingKit.a?

### ISSUE THREAD

**Issue by jameshfisher** [tlenclos/react-native-audio-streaming#20]
I'm trying to follow this instruction:

> In XCode, in the project navigator, select your project. Add libReactNativeAudioStreaming.a and libStreamingKit.a to your project's Build Phases ➜ Link Binary With Libraries

My list under "Link Binary With Libraries" already includes `libReactNativeAudioStreaming.a`. (I'm guessing this is due to the `react-native link react-native-audio-streaming` command I ran.)

However, `libStreamingKit.a` is not there. I clicked the "+", which prompted me to find it. But I don't know where to find it. There is nothing in the repository with this name.

**Comment by jameshfisher** [comment]
I'm trying to find out what `react-native link` command did, but I can't find any documentation for it :( Google sends me to [a page about URLs in react-native](https://facebook.github.io/react-native/docs/linking.html), which I don't think is relevant.

**Comment by jameshfisher** [comment]
[I think the `.a` files are XCode's file extension for static libraries](http://stackoverflow.com/questions/8224035/what-is-a-a-as-libcrypto-a-file) - which I guess means `libStreamingKit.a` is the output of building [this external StreamingKit library](https://github.com/tumtumtum/StreamingKit)

**Comment by jameshfisher** [comment]
I can see [this `StreamingKit/` directory in the `react-native-audio-streaming` repository](https://github.com/tlenclos/react-native-audio-streaming/tree/master/ios/Pods/StreamingKit) - which looks like it's a copy-paste of the external lib? I guess that means it's the version I want to build

**Comment by breadadams** [comment]
Old issue, but I just ran into it now.

Upon running `react-native link` after installing the package, the `libReactNativeAudioStreaming.a` file was included in *Link Binary With Libraries*. However I had to manually add `libStreamingKit.a`.

Had some trouble finding this, so used Xcodes handy search (bottom left) to find the library, and then you just drag it into the list.

![react-native-audio-streaming-find-library](https://cloud.githubusercontent.com/assets/5795227/21535495/22e8f6d6-cd76-11e6-8604-55a4df37b02e.jpg)
